sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/UcfgInstructionFactory.cs

```diff
@@ -149,44 +149,44 @@ public IEnumerable<Instruction> CreateFrom(SyntaxNode syntaxNode)
                 .Concat(BuildAnnotationCall(attributeSyntax, targetOfAttribute, expressionService.GetOrDefault(attributeSyntax)));
         }
 
-        private void ApplyAsTarget(Expression expression, Instruction instruction)
+        private IEnumerable<Instruction> ApplyAsTarget(Expression expression, Instruction instruction)
         {
+            var instructionsToReturn = new[] { instruction };
```

Comment:
Am I misreading, or is _instructionsToReturn_ always empty?
If so, what's the difference between returning _instructionsToReturn_ and _NoInstructions_?